keras/backend/tensorflow_backend.py

```diff
@@ -360,7 +360,7 @@ def to_dense(tensor):
 name_scope = tf.name_scope
 
 
-def variable(value, dtype=None, name=None, constraint=None):
+def variable(value, dtype=None, name=None, constraint=None, sparse=False):
```

Comment:
Why do we need to change tensorflow backend? Please only modify mxnet_backend.py
If you what to pass sparse in Keras K.variable, add a check, so you won't break or change tensorflow backend
```
if backend == mxnet:
 K.variable(sparse = True)
else:
 K.variable()
```

Reply:
done